### DISCUSSION TITLE
六角龙机器人又抽风了

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#3796] [讨论]
看到 Label 被大改了一波，然后邮箱来了一堆邮件……
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/488f6974-1738-4044-905d-32036ffde794)
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/ea0b9952-9fa5-4946-98e4-dba496fe7a9e)
~~六角龙机器人又抽哪门子风了~~

**Comment by MoYuan-CN** [comment]
炸邮箱了的说（

**Reply by wuliaodexiaoluo** [reply]
我这也是，邮箱 99+

**Comment by allMagicNB** [comment]
@LTCatt

**Comment by LTCatt** [comment]
修了（

**Reply by allMagicNB** [reply]
下次改 Label 内容的时候记得先关机器人（

**Comment by Griseo-bh3rd** [comment]
~所以正确的watch方式：在settings->nofitications里面关掉watch的邮件通知，只保留GitHub通知（doge）~

**Comment by iPodtouch6557** [comment]
还是喜欢αβ蛤蟆

**Reply by allMagicNB** [reply]
~~你 gamma～~~
